apps/condo/domains/acquiring/schema/MultiPayment.test.js

```diff
@@ -893,18 +894,36 @@ describe('MultiPayment', () => {
                 expect(multiPayment).toHaveProperty('status', MULTIPAYMENT_DONE_STATUS)
             })
         })
-        test('mobile resident can\'t see his sensitive data in his own MultiPayments', async () => {
+        test('mobile resident can see his public data in his own MultiPayments', async () => {
             const { admin, payments, acquiringIntegration, client } = await makePayerAndPayments()
             const [createdMultiPayment] = await createTestMultiPayment(admin, payments, client.user, acquiringIntegration)
-            // We use raw: true because when using field access, all fields that are not permitted result in error which stops the test
-            let { data: { objs: multiPayments } } = await MultiPayment.getAll(client, {}, { raw: true })
+            let multiPayments = await MultiPaymentPublic.getAll(client, {})
```

Comment:
It's better not to rely on the MultiPayments count
You can use id-query to get the one you previously created.

Reply:
ok, but still using getAll with id in next test to retrieve raw errors